radix-engine/src/system/system_callback.rs

```diff
@@ -871,6 +867,7 @@ impl<C: SystemCallbackObject> KernelCallbackObject for System<C> {
         let limits_module = { LimitsModule::from_params(system_parameters.limit_parameters) };
 
         let costing_module = CostingModule {
+            current_depth: 0, // TODO: Is it correct to assume that the current depth is zero here?
```

Comment:
I think this was missed? We should resolve this TODO - either commenting why we can assume this, or handling it properly?

Seems reasonable to me that `init` sets `current_depth` to `0` though 👍

Reply:
Yup I missed this, I've removed the TODO comment and will add a comment explaining why we set it to zero.